pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -23,6 +23,29 @@
   # are, so long as it provides some builtins.
   doFakeLibgcc ? stdenv.hostPlatform.isFreeBSD,
 
+  # Whether to build the set of __atomic_* routines that would typically
+  # be provided by libatomic in gcc environments. Although this has
+  # always been available, we only enable this by default on LLVM >= 13
+  # since that's when the standalone build became available; GCC's
+  # libatomic should be used in other cases.
+  withAtomics ?
+    (stdenv.hostPlatform.useLLVM or false)
```

Comment:
We don't need the `or false` since `lib/systems/default.nix` adds it in.

Reply:
ah, good to know. we do `stdenv.hostPlatform.useLLVM or false` everywhere in this file so I assumed I was missing something